account_chatter/views/account_move_views.xml

```diff
@@ -0,0 +1,26 @@
+<odoo>
+    <record id="view_move_form_inherit" model="ir.ui.view">
+        <field name="name">account.move.form.inherit</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="groups_id" eval="[(4, ref('account_chatter.group_show_account_chatter_notifications'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//sheet" position="after">
+                <div class="oe_chatter">
+                    <field name="message_follower_ids"/>
```

Comment:
I don't get this change. Could you explain, please?

Reply:
This change is to show these fields only with the group ```account_chatter.group_show_account_chatter_notifications```. Why by default the chatter is hidden from all users

Reply:
ok, But why the class d-none and that stuff?

Wouldn't be simpler to add the fields and add the desired group as an attribute?

Reply:
> ok, But why the class d-none and that stuff?

Because the chatter must be hidden by default, if the user does not belong to the group

> Wouldn't be simpler to add the fields and add the desired group as an attribute?

Because the fields (chatter) are already shown by default in odoo. This module now hides them by default and they are only visible through the group

Reply:
> Because the fields (chatter) are already shown by default in odoo. This module now hides them by default and they are only visible through the group

That's why I ask, why not simply adding the attribute group="group_can_see_chatter"? That would show the chatter only for members of that group. It doens't need to be used on the fields themselves, it could be applied in the whole chatter dif.

Reply:
Yes, this was my first option but it did not work with the attribute group, neither for this fields nor for the div, reading I found this forum https://www.odoo.com/es_ES/forum/help-1/visibility-of -oe-chatter-only-form-employee-manager-how-to-141797